build.gradle

```diff
@@ -31,18 +31,27 @@ plugins {
     id "org.jlleitschuh.gradle.ktlint" version "${gradleKtlintVersion}"
 }
 
-allprojects {
-    repositories {
-        google()
-        mavenCentral()
-    }
+dependencies {
```

Comment:
Including a couple documentation related fixes. These have been in the `documentation` branch for a while but I forgot to bring them back into master